notation/verify.ts

```diff
@@ -0,0 +1,41 @@
+import * as path from 'path';
+import * as taskLib from 'azure-pipelines-task-lib/task';
+
+import { getArtifactReferences } from './lib/variables';
+import { getConfigHome } from './lib/fs';
+
+export async function verify(): Promise<void> {
+    const artifactRefs = getArtifactReferences();
+    const trustpolicy = taskLib.getInput('trustpolicy', true) || '';
+    const truststore = taskLib.getInput('truststore', false) || '';
+    const configHome = getConfigHome();
+    const allowReferrerAPI = taskLib.getBoolInput('allowReferrersAPI', false);
+    const debug = taskLib.getVariable('system.debug')
+
+    // copy the trustpoliy to notation config dirctory
+    const trustPolicyPath = path.join(configHome, 'notation', 'trustpolicy.json');
+    taskLib.cp(trustpolicy, trustPolicyPath, '-f');
+
+    // copy the trust store directory to notation config directory
+    const trustStorePath = path.join(configHome, 'notation')
+    taskLib.rmRF(trustStorePath);
```

Comment:
Btw, this is removing the whole notation folder right? Should be:
```suggestion
    const trustStorePath = path.join(configHome, 'notation', 'truststore')
    taskLib.rmRF(trustStorePath);
```

Reply:
updated.